components/image-builder-bob/pkg/builder/config.go

```diff
@@ -36,7 +33,6 @@ func GetConfigFromEnv() (*Config, error) {
 		BaseRef:            os.Getenv("BOB_BASE_REF"),
 		BaseContext:        os.Getenv("THEIA_WORKSPACE_ROOT"),
 		BuildBase:          os.Getenv("BOB_BUILD_BASE") == "true",
-		BaseLayerAuth:      os.Getenv("BOB_BASELAYER_AUTH"),
```

Comment:
I couldn't find any record of this var being used so I removed it to tidy it up. Happy to add it back in if it's important.